website/data/menu/main.yml

```diff
@@ -1153,8 +1170,13 @@ main:
             ref: "/terminal/forex/load"
           - name: candle
             ref: "/terminal/forex/candle"
-      - name: resources
-        ref: "/terminal/resources"
+          - name: from
+            ref: "/terminal/forex/from"
+          - name: oanda
+            ref: "/terminal/forex/oanda"
+            sub:
```

Comment:
There are a lot of command missing here it seems:
<img width="1421" alt="Screenshot 2022-03-16 at 10 09 37" src="https://user-images.githubusercontent.com/25267873/158566666-ddf3bfd3-6b88-454f-9c93-fb33167dc725.png">